src/Altinn.Platform/Altinn.Platform.Storage/Storage/Program.cs

```diff
@@ -46,22 +45,21 @@ public static void Main(string[] args)
             {
                 webBuilder.UseStartup<Startup>()
 
-                  // Parameters required for integration with SBL in local development
-                   /* .UseKestrel()
-                    .UseContentRoot(Directory.GetCurrentDirectory())
-                    .UseIISIntegration()*/
+                    // Parameters required for integration with SBL in local development
+                    ////.UseKestrel()
```

Comment:
I'm not sure we need these any longer.. Not quite sure why, but one of our recent changes has made the need for this obsolete.

Reply:
I've confirmed that we no longer need this lines to integrate with SBL, so they may be deleted. lines 49-52

Reply:
Deleted the commented lines.